itest/lnd_channel_graph_test.go

```diff
@@ -420,6 +424,24 @@ func testNodeAnnouncement(ht *lntest.HarnessTest) {
 	assertAddrs(nodeUpdate.Addresses, advertisedAddrs...)
 }
 
+// testGetChannelInfoNotFound verifies that a missing channel reports an
+// appropriate gRPC status code when calling GetChanInfo
+func testGetChannelInfoNotFound(ht *lntest.HarnessTest) {
+	ctx, cancel := context.WithTimeout(context.Background(), lntest.DefaultTimeout)
+	defer cancel()
+
+	alice := ht.NewNode("Alice", nil)
+	_, err := alice.RPC.LN.GetChanInfo(ctx, &lnrpc.ChanInfoRequest{
```

Comment:
Can we create a `GetChanInfoAssertErr` instead to keep the format consistent? sth like,
https://github.com/lightningnetwork/lnd/blob/3707b1fb7032d98dc2a8e881d3ae83e61fbc35e3/lntest/rpc/lnd.go#L366-L390